common/src/main/java/no/difi/meldingsutveksling/config/dpi/PrintSettings.java

```diff
@@ -0,0 +1,40 @@
+package no.difi.meldingsutveksling.config.dpi;
+
+public class PrintSettings {
+    private ReturnType returnType;
+    private InkType inkType;
+    private ShippingType shippingType;
+
+    public PrintSettings(ReturnType returnType, InkType inkType, ShippingType shippingType) {
+        this.returnType = returnType;
+        this.inkType = inkType;
+        this.shippingType = shippingType;
+    }
+
+    public PrintSettings() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)